### PR TITLE
fix(sdk): Redesign Interactive Question Notebook button

### PR DESCRIPTION
**Pull request by oisincoveney**
Modifies the Editor button on InteractiveQuestion to match the new design. Here's a demo

https://github.com/user-attachments/assets/8f5cfe81-8a6b-4207-b54a-79acdb95109e

### CODE REVIEW

File: enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx [hunk 1]
```diff
@@ -16,10 +17,14 @@ export const EditorButton = ({
       question,
       isActionListVisible: true,
     }) && (
-      <QuestionNotebookButton
-        isShowingNotebook={isOpen}
-        setQueryBuilderMode={onClick}
-      />
+      <ActionIcon
+        size="lg"
+        color="var(--mb-color-brand)"
```
Comment (heypoom) [inline enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx:22]: Would `color="brand"` work here?

**Review by heypoom — APPROVED** [review]

**Comment by heypoom** [comment]
I think the notebook button does look out of place with the header though, could you check with Maz on the design? For example, the inactive color is solid black, while the chart type picker is light grey.

CleanShot 2567-11-20 at 17 15 12@2x

**Comment by oisincoveney** [comment]
> I think the notebook button does look out of place with the header though, could you check with Maz on the design? For example, the inactive color is solid black, while the chart type picker is light grey.
> 
> CleanShot 2567-11-20 at 17 15 12@2x

nice catch! I've fixed it and it looks like this:

image

File: enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx (hunk 1 above)
Comment (oisincoveney) [reply enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/EditorButton.tsx]: yes!

**Pull request merged**